pkg/extensions/sync/signatures.go

```diff
@@ -608,3 +608,15 @@ func getCosignTagFromImageDigest(digestStr string) string {
 
 	return digestStr
 }
+
+func getNotationManifestsFromOCIRefs(ociRefs ispec.Index) []ispec.Descriptor {
+	notaryManifests := []ispec.Descriptor{}
+
+	for _, ref := range ociRefs.Manifests {
+		if ref.ArtifactType == notreg.ArtifactTypeNotation {
+			notaryManifests = append(notaryManifests, ref)
```

Comment:
hmm, why is this not covered?

should be covered from all the notation tests, right?

Reply:
nvm, it's ok, is not covered because of the order of signing in tests (first cosign then notation)